template/java8/model/src/test/java/RequestTest.java

```diff
@@ -10,43 +13,43 @@
 	@Test
 	public void testSingleRequestParameterGetSet()
 	{
-		Request request = new Request(null,null,"testParam=testParamValue", null);
+		Request request = new Request("", null, "testParam=testParamValue", null);
```

Comment:
Why not keeping it null?

Reply:
I did some back and forth here, forgot to change it back. Will do.

Reply:
Ah, yes I did this because of conflicting constructors when using `null`. Changed to `(String)null`